editors/code/src/client.ts

```diff
@@ -126,16 +127,16 @@ export async function createClient(
                         }
```

Comment:
github doesn't let me mark them but the regex matching stuff breaks with the colors, if you toggle `previewRustcOutput` you can observethe diagnostics no longer appear in the client

Reply:
Even if it is already broken, I think you're right there's a good chance this would break it. The simplest solution is probably to strip out the escape codes once more before doing the regex match

Reply:
No this seems fine on master, so my initial assumption seems correct